kickstarts/partials/post/vagrant.ks.erb

```diff
@@ -0,0 +1,23 @@
+# Add a user "vagrant" with password "vagrant" and a well known public key.
+# Note this is how Vagrant works. You are assumed not to use a Vagrant box on
+# a publicly acessible network.
```

Comment:
This feels like a big assumption.  I'm not familiar with vagrant to understand why this is needed.